config_defaults/subtests/docker_cli/negativeusage.ini

```diff
@@ -70,24 +70,6 @@ subarg = -e,PATH=/tmp,%%(FQIN)s,true
 stderr = exec\: \"true\"\: executable file not found in \$PATH
 extcmd = 127
 
-[docker_cli/negativeusage/iv5]
-subcmd = run
-subarg = --tmpfs,/dev,%%(FQIN)s
-stderr =  no such file or directory
-extcmd = 127
-
-[docker_cli/negativeusage/iv6]
-subcmd = run
-subarg = --tmpfs,:rw,%%(FQIN)s
-stderr =  open /proc/self/task/1/attr/exec: no such file or directory
-extcmd = 125
-
-[docker_cli/negativeusage/iv7]
-subcmd = run
-subarg = --tmpfs,.,%%(FQIN)s
-stderr =  Message: Failed to open /dev/null - open /dev/null: permission denied
-extcmd = 125
```

Comment:
>The iv7 test should be extra skipped

Only if it's called iv7. I tried reenabling it, but called it `iv5` because the gap offends me and will confuse future maintenance programmers. That resulted in a total barf: `DockerTestError: Sub-subtest(s) set(['docker_cli/negativeusage/iv7']) referenced in control include, exclude, and/or subthings but not defined in subsubtests configuration option: ['op1', 'op2', 'op3', 'if1', 'ov1', 'iv1', 'iv3', 'iv4', 'iv5', 'ip1']` (yes, I rebased on to master with pr655). So I disabled the bugzilla check because I wanted to actually test the code.

Bottom line: I'm not comfortable with band-aiding by disabling the test, because once (if) that BZ gets closed, we will still have a situation in which docker 1.10 exits 125, 1.12 126, and docker-FUTURE exits with TBD. And tests will break, and we'll have to scramble to find a solution that works on all versions of docker.

I totally agree that a test would be nice. I want to write that test *on our terms*, on our time, not as an emergency dictated by failing tests whenever that BZ happens to get closed. Does that make sense?